lib/elixir/lib/enum.ex

```diff
@@ -2175,11 +2213,39 @@ defmodule Enum do
     end
   end
 
-  def min_max(enumerable, empty_fallback) when is_function(empty_fallback, 0) do
+  def min_max(enumerable, empty_fallback)
+      when is_function(empty_fallback, 0) do
+    min_max(enumerable, &</2, empty_fallback)
```

Comment:
Shouldn't it be:

```suggestion
    min_max(enumerable, &<=/2, empty_fallback)
```

Otherwise we may have a change in behaviour here? Although we do have tests verifying the property...

Reply:
in min_max_by it uses `&</2`

Reply:
This is weird. Does it fail for `<=`?

Reply:
changing it fails one test:
```
     Assertion with === failed                                             
     code:  assert Enum.min_max([1, 1.0]) === {1, 1}                       
     left:  {1.0, 1}                                                       
     right: {1, 1}            
 ```